cmake/mcuboot.cmake

```diff
@@ -71,7 +71,8 @@ function(zephyr_mcuboot_tasks)
   endif()
 
   # Basic 'west sign' command and output format independent arguments.
-  set(west_sign ${WEST} sign --quiet --tool imgtool
+  set(west_sign ${WEST} sign ${CONFIG_MCUBOOT_CMAKE_WEST_SIGN_PARAMS}
```

Comment:
Does `separate_arguments()` need to be used on this as the code below uses?

Reply:
OK, so I have checked documentation and as far as I understand `separate_arguments` would convert the string to list of command and arguments. @nordicjm Can you help me with how should I use the function here?

Reply:
```
  separate_arguments(west_sign_extra UNIX_COMMAND ${CONFIG_MCUBOOT_CMAKE_WEST_SIGN_PARAMS})
  set(west_sign ${WEST} sign ${west_sign_extra}
```
Should work